tyk-pro/templates/deployment-tib.yaml

```diff
@@ -59,7 +59,8 @@ spec:
           - name: TYK_IB_SECRET
             value: "{{ .Values.tib.secret }}"
           - name: TYK_IB_HTTPSERVEROPTIONS_USESSL
-            value: "{{ .Values.tib.useSSL }}"
+          # Enable backward compatibility to chart 0.8.1.
+            value: "{{ or .Values.tib.tls .Values.tib.useSSL }}"
```

Comment:
i think it's missing `if` keyword

Reply:
ugggh. Thank you!

Reply:
This is just an or there shouldn't be an `if` before it